_posts/2018-01-18-From-Manual-to-Automatic.md

```diff
@@ -0,0 +1,111 @@
+
+- Cumulative coverage of previously developed features, increasing defect detection and reducing bug fixing costs; 
+- Reproducibility by ensuring software stability and reduced market times and costs; 
+- More effective resource management, resulting in increased productivity. 
+
+To better demonstrate the advantages of test automation when compared to manual testing, our team conducted a basic study with five manual and automatic tests on an internal prototype webapp. As provided in the Appendix I of this article, it has been considered the average metrics for the human resources effort costs for implementing both the manual tests and the automation tests. Because it has been assumed that the maintenance costs were equivalent in both approaches, it has been decided that these were not to be included in the study.
+Automation scripting times were considerably higher than manual test specification, but get progressively this starting discrepancy has been diluted as the number of test executions cycles increases in both scenarios. For this specific study we've conclude that automatic testing becomes worthwhile after the 36th execution cycle when compared with the same number of manual executions cycles, as shown in the following graphic: 
+
+<br><img src="http://enear.github.io./public/qa-article-images/exec_by_cost_graphic.png"><br>
+
+
+If we were to consider three daily test campaigns cycles, software automation testing would start to compensate after twelve days. 
+
+## How to assess what should be automated 
+
+As previously stated, today’s market contains increasingly complex applications that are constantly improving to offer more features to the end consumer. Between other factors when combined with agile development methodologies it is then possible that these continually mutating products achieve lower *time to market* business metrics, however, both existing and new features need always to be tested to ensure that they meet the expected product quality standards.
+Test automation is then a powerful software development ally since it promotes and maintains quality patterns throughout the product´s lifecycle culminating in swifter results in the development cycles without sacrificing the team with quality-related overheads<sup>[2]</sup>.
+
+Elegible product features that are candidates for test automation are usually those that were previously developed, the one that are considered stable or that have been already manually tested and with known results. On the other hand, unstable, temporary, incomplete features, or those whose automation is so complex, expensive, time-consuming and with high maintenance times are usually left out. This is due to the high initial costs of test automation which are only compensated and supported by successive regression test cycles being a medium for long product life cycles<sup>[3]</sup>.
```

Comment:
`Elegible product features that are candidates for test automation are usually those that were previously developed, the one that are considered stable or that have been already manually tested and with known results.`
->
`Eligible product features and candidates for test automation are usually those that were previously developed, those that are considered stable or that have been already manually tested and with known results.`